Kernel/KerrGeoOrbit.m

```diff
@@ -19,7 +19,7 @@
 Begin["`Private`"];
 
 
-(* ::Section::Closed:: *)
+(* ::Section:: *)
```

Comment:
These changes to section open/closed status shouldn't be included. Similarly for the others in this file